tests/phpunit/tests/block-library/commentTemplate.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * Comment Template block tests
+ *
+ * @package WordPress
+ * @subpackage Block Library
+ * @since 6.0.0
+ */
+
+/**
+ * Tests for the Comment Template block.
+ *
+ * @since 6.0.0
+ *
+ * @group block-library
+ */
+class Test_Block_Library_CommentTemplate extends WP_UnitTestCase {
```

Comment:
I've defined a new `@subpackage` and `@group` for this test, as this is the first one we have for a specific block (as far as I saw).

I've also modified the class name to follow the same format as the other test files.

Reply:
Let me know if this file should be placed under a different folder/subpackage/etc. 🙂